tox.ini

```
@@ -0,0 +1,17 @@
+[tox]
+envlist = py35-django22{,-user_orgs}, py38-django22{,-user_orgs}
+
+[testenv]
+setenv =
+    DJANGO_SETTINGS_MODULE = test_settings
+    PYTHONPATH = {toxinidir}
+    TAHOE_SITES_USE_ORGS_MODELS = False
+    use_orgs: TAHOE_SITES_USE_ORGS_MODELS = True
+
+deps =
+    django22: Django>=2.2,<3.0
+    -rrequirements/tests.txt
```

That's the most reliable method of multiplexing test dependencies. It's better than having the tests inside the repo.

```suggestion
    withorgs: https://github.com/appsembler/edx-organizations/archive/5.2.0-appsembler13.tar.gz
    noorgs: edx-organizations==5.2.0
    -rrequirements/tests.txt
```